hal/inc/concurrent_hal.h

```diff
@@ -132,4 +181,3 @@ void os_thread_scheduling(bool enabled, void* reserved);
 #endif
```

Comment:
It's a C API - there are no default values in C. As soon as we raise the bar to having everything C++ then we can do this (and fix some other long standing issues, like having to use #defines for pin constants.)